modules/decima-model/src/main/java/com/shade/decima/model/rtti/messages/hfw/HFWIndexArrayResourceHandler.java

```diff
@@ -18,20 +18,31 @@
     @Type(name = "IndexArrayResource", game = GameType.HFW),
 })
 public class HFWIndexArrayResourceHandler implements MessageHandler.ReadBinary {
+    private static final int[] unkArray = new int[]{
+        0, 0x10, 0x20, 0x40, 0x20, 0x40, 0x60, 0x80,
+        0x8, 0x10, 0x20, 0x40, 0x60, 0x80, 0x20, 0x40, 0x60, 0x80,
+        0x8, 0x10, 0x20, 0x40, 0x20, 0x20, 0x40, 0x20, 0x20, 0x40, 0x20, 0x20, 0x40, 0x20,
+    };
```

Comment:
Probably an enumerator of some sort. Was it scavenged from the binary or constructed manually?